### PR TITLE
T1112301

### PR DESCRIPTION
**Pull request by ScottDowne**

### CODE REVIEW

File: simple-snippet/snippet.html [hunk 1]
```diff
@@ -90,8 +90,15 @@
...
-  </style>
+        #mofo-eoy-fundraiser-{{ snippet_id }} .thumbnail-img {
+          float: left;
+          margin: 5px;
+        }
 
+        img[src=""] { display: none }
+
+  </style>
+    <img class="thumbnail-img" src="{{ thumbnailImg }}" />
```
Comment (adamlofting) [inline simple-snippet/snippet.html:101]: As this updates the existing snippet template, do we need some logic to only show the `` if there is a `{{ thumbnailImg }}`? 

(We'll still have the existing variants running live without an image, and if the image testing doesn't improve the performance we'll keep the text only version as our default)

Comment (ScottDowne) [inline simple-snippet/snippet.html:101]: yeah, there is a css rule `img[src=""] { display: none }`

That'll hide any images with no source.

Comment (adamlofting) [inline simple-snippet/snippet.html:101]: That's neat :) I hadn't seen that done like that.

Comment (ScottDowne) [inline simple-snippet/snippet.html:101]: It was k88hudson's idea

**Comment by mmmavis** [comment]
r+

**Pull request closed without merging**